src/components/Icon/index.tsx

```diff
@@ -39,8 +39,22 @@ import type { StyledProps } from '@types'
 
 export type IconType = keyof typeof ICON_TYPES
 export interface IconProps extends HTMLAttributes<HTMLOrSVGElement> {
+  /**
+   * Icon의 사이즈를 정합니다.
```

Comment:
```suggestion
   * Icon의 크기를 정합니다.
```
요기 한글로 의미 전달해도 좋을 것 같아요!

Reply:
[요기](https://github.com/price-offer/offer-design-system/pull/82/commits/d25c1a5824bdc6dc9f826b2f031f9ebaa2c886d7)에 반영 완료! 🥰